deeppavlov/models/classifiers/proba2labels.py

```diff
@@ -70,7 +70,7 @@ def __call__(self, data: Union[np.ndarray, List[List[float]], List[List[int]]],
             return [list(np.where(np.array(d) > self.confident_threshold)[0])
                     for d in data]
         elif self.max_proba:
-            return [[np.argmax(d)] for d in data]
+            return [np.argmax(d) for d in data]
```

Comment:
there is something weird with this `__call__()` method typing.

Reply:
fixed